libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -132,6 +246,69 @@ def add_texts(
         # Insert the embedded texts in the vector store table.
         return self._insert_embeddings(texts, embedded_texts, metadatas, ids)
 
+    def drop(self) -> None:
+        """Drops every table created during initialization of vector store."""
+        logging.info("Dropping vector store: %s.", self.table_name)
+        try:
+            with Session(bind=self._bind) as session:
+                # Drop all the tables associated with the session bind.
+                Base.metadata.drop_all(session.get_bind())
+        except ProgrammingError as e:
+            logging.error(f"Unable to drop vector store.\n {e.__cause__}.")
+
+    def _search_store(
+        self, embedding: List[float], k: int, filter: Optional[dict] = None
+    ) -> List[Any]:
+        try:
+            # The filter variable will be passed as an argument to the filter function
+            # filter_clause = self._create_filter_clause(filter)
+            with Session(self._bind) as session:
+                results = (
+                    session.query(
+                        _embedding_store,
+                        text(VECTOR_DISTANCE_QUERY).bindparams(
+                            bindparam(
+                                DISTANCE_STRATEGY,
+                                self.distance_strategy,
+                                literal_execute=True,
+                            ),
+                            bindparam(
+                                EMBEDDING, json.dumps(embedding), literal_execute=True
+                            ),
+                        ),
+                    )
+                    .filter()
+                    .order_by(asc(text("distance")))
```

Comment:
We should check how this works for case-sensitive collation

Reply:
Done. Replied earlier comment.